docs/architecture/PROCESS.md

```diff
@@ -0,0 +1,56 @@
+# ADR Creation Process
```

Comment:
ported it over from: https://github.com/cosmos/cosmos-sdk/blob/9833bf14c1e1388feca724691fe02f0fdac3271c/docs/architecture/PROCESS.md

Not sure why it was missing.